src/commands/orgs/list.ts

```diff
@@ -4,40 +4,17 @@ import { MobileCenterClient, models, clientRequest } from "../../util/apis";
 
 const debug = require("debug")("mobile-center-cli:commands:orgs:list");
 import { inspect } from "util";
+import { getOrgsNamesList } from "./lib/org-users-helper";
 
 @help("Lists organizations in which current user is collaborator")
 export default class OrgListCommand extends Command {
   async run(client: MobileCenterClient): Promise<CommandResult> {
     // every user is a collaborator of it's own group and of zero or more external groups
-    const orgs = await out.progress("Loading list of organizations...", this.getOrgsNamesList(client));
+    const orgs = await out.progress("Loading list of organizations...", getOrgsNamesList(client, debug));
     const table = orgs.map((names) => [names.displayName, names.name, names.origin]);
 
     out.table(out.getCommandOutputTableOptions(["Display Name", "Name", "Origin"]), table);
 
     return success();
   }
-
```

Comment:
Why did you make this refactoring - is it part of the feature you're implementing somehow (seems odd that a testcloud command would need to touch the orgs commands)?

Reply:
This was after some discussions with @skovsboll - I needed to determine if the current run was from a user or an organization (so that I could generate and report the correct URL to the user). Unfortunately there seemed to be no easy way to do this so this was the best option we came up with :)

I moved the list so it was accessible to this new functionality.

Happy to change it if there's a better way!